source/extensions/common/wasm/context.cc

```diff
@@ -648,6 +648,8 @@ Http::HeaderMap* Context::getMap(HeaderMapType type) {
   }
 }
 
+#define GET_FROM_ROOT(x) isRootContext() ? x : root_context_->x
```

Comment:
You could just make this an inline function.  Up2U

Reply:
Done, PTAL.